pkgs/development/tools/lazygit/default.nix

```
@@ -11,14 +11,14 @@ buildGoPackage rec {
   src = fetchFromGitHub {
     owner = "jesseduffield";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "0zynw5gr96a59x1qshzhhvld883ndf1plnw6l9dbhmff0wcfv6l1";
+    rev = "963c034b481e6676656e981cbc8e2f923cc1ac41";
```

```suggestion
    rev = "v${version}";
```

Please keep it that way. When the package is version 0.10.6 we actually want that.
If you want to add some patches on top that's okay.

But they seem to make new releases very quickly, so we can just update it again, then, to include your favorite patches from master :)